scripts/process/merge_centroids_wu.R

```diff
@@ -0,0 +1,14 @@
+process.merge_centroids_wu <- function(viz) {
+  deps <- readDepends(viz)
+  wu_data_15_simple <- deps[["wu_data_15_simple"]]
+  centroids_topo <- geojsonio::topojson_read(as.viz(viz$depends$centroids_topo)$location, stringsAsFactors=FALSE) # bypass readDepends for topojson
```

Comment:
why do we need to bypass readDepends for topojson?

Reply:
we could add a readData function for topojson specifically, but we don't have one now, and I don't feel like it's worth it given that vizlab doesn't break right now and it's just one line of code to do it here. Still, I've added an issue to vizlab at https://github.com/USGS-VIZLAB/vizlab/issues/346, so maybe we'll have a readData.topojson function in the future.